build-deploy-platform-services/Jenkinsfile-deploy-platformservices

```diff
@@ -1,25 +1,24 @@
 #!groovy
 import groovy.json.JsonOutput
+
 node {
 
 		withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: env.JENKINS_CREDENTIAL_ID, passwordVariable: 'PWD', usernameVariable: 'UNAME']]){
 			
 		echo "Build to deploy all Platform Services"
 		
-		def platform_srvc_build_job = "job/build-deploy-platform-service/buildWithParameters?token=bld-plat-srvs-71717&admin_group=admin_group&region=" + params.region;
-
-		def var_job_url = JenkinsLocationConfiguration.get().getUrl() + platform_srvc_build_job;
+		def platform_service_build_job = "job/build-deploy-platform-service/buildWithParameters?token=bld-plat-srvs-71717&admin_group=admin_group&region=" + params.region;
+		def job_url = JenkinsLocationConfiguration.get().getUrl() + platform_service_build_job;
 		
 		def timeout_seconds = 55;
-		
-		echo "Job URL:: " + var_job_url;
-			
+
+		echo "Job URL:: " + job_url;	
 		def platform_services = ["cognito-authorizer", "create-serverless-service", "delete-serverless-service", "cloud-logs-streamer", \
 								"is-service-available",  "platform_login", "platform_logout", "platform_services", "platform_usermanagement", \
 								"platform_logs", "platform_events", "platform-services-handler", "platform_email"] 
 
 		platform_services.each {
-			sh "curl -X GET -k -v -u \"$UNAME:$PWD\"  \"${var_job_url}&service_name=${it}\""       
+			sh "curl -X GET -k -v -u \"$UNAME:$PWD\"  \"${job_url}&service_name=${it}\""       
```

Comment:
thanks for fixing these inconsistencies in naming variables

Reply:
np